voltdbclient/url.go

```diff
@@ -25,9 +26,23 @@ func parseURL(src string) (*url.URL, error) {
 			}
 		}
 	} else {
-		if u.Port() == "" {
+		if getPort(u.Host) == "" {
 			u.Host = u.Host + ":21212"
 		}
 	}
 	return u, nil
 }
+
+func getPort(host string) string{
```

Comment:
I agree with @gernest . This looks like the same as the one in the Go standard lib, which is used by `URL.Port()`. It's not necessary to copy the implementation here.